02Lesson/01Project/index.html

```diff
@@ -1,9 +1,33 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-</head>
-<body>
-
-</body>
+    <head>
+          <meta charset="utf-8">
+      <title>
+        Allisons Custom Dreamcatchers
+      </title>
+    </head>
+    <body>
+      <img src="http://cdn1.dreamcatchercomi.netdna-cdn.com/images/A/dc775-purple-soul-connection-with-swarovski-crystal-dreamcatcher.jpg" alt="Dreamcatcher image">
+        <h1>
+          Allison's Custom Dreamcatchers
+        </h1>
+          <p>
```

Comment:
Indentation: After closing `h1` tag, next element needs to be on the same level.